packages/sources/react/src/components/VtmnTextInput/VtmnTextInput.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import clsx from 'clsx';
+
+import '@vtmn/css-text-input';
+
+export interface VtmnTextInputProps
+  extends React.InputHTMLAttributes<HTMLInputElement> {
+  /**
+   * Label text linked to the input
+   * @type {string}
+   */
+  label: string;
```

Comment:
It could be nice to follow the same props names that we already used in `VtmnTextInput.svelte` proposed by @thollander.
So for this example it's `labelText` instead of `label`